src/legacy/core_plugins/kbn_vislib_vis_types/public/editors/point_series.tsx

```diff
@@ -77,9 +79,61 @@ function PointSeriesOptions(props: VisOptionsProps<BasicVislibParams>) {
             }
           />
         )}
+
+        <SwitchOption
+          label={i18n.translate('kbnVislibVisTypes.editors.pointSeries.thresholdLine.show', {
+            defaultMessage: 'Show threshold line',
+          })}
+          paramName="show"
+          value={stateParams.thresholdLine.show}
+          setValue={(paramName, value) =>
+            setValue('thresholdLine', { ...stateParams.thresholdLine, [paramName]: value })
+          }
+        />
+
+        {stateParams.thresholdLine.show && (
+          <NumberInputOption
+            label={i18n.translate('kbnVislibVisTypes.editors.pointSeries.thresholdLine.valueLabel', {
+              defaultMessage: 'Threshold value',
+            })}
+            paramName="value"
+            value={stateParams.thresholdLine.value}
+            setValue={(paramName, value) =>
+              setValue('thresholdLine', { ...stateParams.thresholdLine, [paramName]: value })
+            }
+          />
+        ) && (
+          <NumberInputOption
+            label={i18n.translate('kbnVislibVisTypes.editors.pointSeries.thresholdLine.widthLabel', {
+              defaultMessage: 'Line width',
+            })}
+            paramName="width"
+            min="1"
+            value="1"
+            value={stateParams.thresholdLine.width}
+            setValue={(paramName, value) =>
+              setValue('thresholdLine', { ...stateParams.thresholdLine, [paramName]: value })
+            }
+          />
+        ) && (
+          <SelectOption
+            label={i18n.translate('kbnVislibVisTypes.editors.pointSeries.thresholdLine.style', {
+              defaultMessage: 'Line style',
+            })}
+            options={['full', 'dashed', 'dot-dashed']}
```

Comment:
Those strings need to be properly i18n'ed (it's also one of the TypeScript errors, that they are not in the format of `{ value: string; text: string }`.